tern/analyze/common.py

```diff
@@ -452,3 +453,61 @@ def get_os_style(image_layer, binary):
                     package_manager=binary,
                     package_format=image_layer.pkg_format,
                     os_list=image_layer.os_guess), 'info'))
+
+
+def check_git_src(dockerfile_path, src_path):
+    '''Given the src_path and the dockerfile path, return the git
+    repository name and sha information in the format of string.
+    Currently we only consider the following situation:
+    - target_git_project
+        - dir1
+        - dir2/dockerfile
+    So we only use dockerfile_path to find the git repo info.'''
+    # get the path of the folder containing the dockerfile
+    dockerfile_folder_path = os.path.dirname(dockerfile_path)
+    # locate the top level directory
+    path_to_toplevel = get_git_toplevel(dockerfile_folder_path)
+    # get the path of the target folder or file
+    logger.debug('looking into path: %s for git repo.', path_to_toplevel)
+    # add this line to avoid linting check
+    logger.debug('src_path of ADD command: %s.', src_path)
```

Comment:
I find the `src_path` of ADD command is not used when looking for the toplevel folder. So I add this line to avoid linting check. `src_path` may be useful when considering other corner situations.

Reply:
So if you remove `src_path` these two lines can be removed as well.